tests/array.rs

```diff
@@ -1891,3 +1891,119 @@ fn array_macros() {
     let empty2: Array<f32, Ix2> = array![[]];
     assert_eq!(empty2, array![[]]);
 }
+
+#[cfg(test)]
+mod array_cow_tests {
+    use super::*;
+    use ndarray::Data;
+
+    fn is_content_identical<A, S1, S2, D>(arr1: &ArrayBase<S1, D>, arr2: &ArrayBase<S2, D>) -> bool
+        where A: Clone + PartialEq,
+              S1: Data<Elem=A>,
+              S2: Data<Elem=A>,
+              D: Dimension
+    {
+        arr1.iter().zip(arr2.iter()).all(|(x1, x2)| x1 == x2)
+    }
+
+    #[test]
+    fn test_is_view() {
+        let arr: Array<i32, Ix2> = array![[1, 2], [3, 4]];
+        let arr_cow = ArrayCow::<i32, Ix2>::from(arr.view());
+        assert!(arr_cow.is_view());
+        let arr_cow = ArrayCow::<i32, Ix2>::from(arr.clone());
+        assert!(!arr_cow.is_view());
+    }
+
+    #[test]
+    fn test_is_owned() {
```

Comment:
It's only a suggestion, but I would merge `test_is_view` and `test_is_owned` because one is simply the reverse of the other,